NOTICE

```diff
@@ -35,6 +35,33 @@ the following terms:
   Copyright 2015 WebAssembly Community Group participants
   Apache License, Version 2.0 (see LICENSE file)
 
+* Arm Optimized Routines: https://github.com/ARM-software/optimized-routines
+  Copyright (c) 2019, Arm Limited.
+  SPDX-License-Identifier: MIT
```

Comment:
The copyright is also in the license, so I guess it's fine to omit those two lines and start with the license right away.